.project-creation/templates/requirements.in

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+##########################################
+# Import src and test requirements
+##########################################
+-c app/requirements.txt
+-c app/tests/requirements.txt
+##########################################
+# Development Tools Packages
+##########################################
+grpcio-tools==1.57.0
```

Comment:
```suggestion
grpcio-tools==1.59.0
```

Reply:
Is there any reason why we need to do this **now**? We should really release this PR at some point 😅

Reply:
I suggested this to get it in sync with the (grpcio) versions used in the other places. Serhii pointed a hint that grpcio and grpcio-tools versions should be in sync. I've no clue how relevant this is actually.

Reply:
If it's tested and working in this constellation, we could keep it for now ...